common/persistence/cassandra/cassandraMetadataPersistenceV2.go

```diff
@@ -408,6 +426,14 @@ func (m *cassandraMetadataPersistenceV2) GetDomain(request *p.GetDomainRequest)
 	replicationConfig.Clusters = p.DeserializeClusterConfigs(replicationClusters)
 	replicationConfig.Clusters = p.GetOrUseDefaultClusters(m.currentClusterName, replicationConfig.Clusters)
 
+	var failoverConfig *p.DomainFailoverConfig
+	if failoverStartTime > 0 && failoverTimeout > 0 {
+		failoverConfig = &p.DomainFailoverConfig{
+			StartTime: failoverStartTime,
+			Timeout:   failoverTimeout,
+		}
+	}
+
```

Comment:
Looks if is not necessary. maybe just 
```
&p.DomainFailoverConfig{
			StartTime: failoverStartTime,
			Timeout:   failoverTimeout,
		}
```
in line 445

Reply:
Updated to use failover end time. Here if it is nil, it means there is no graceful failover happening